src/tlo/methods/skeleton.py

```diff
@@ -148,6 +149,16 @@ def report_daly_values(self):
         """
         raise NotImplementedError
 
+    def report_prevalence(self):
+        """
+        This must send back a pd.Series or pd.DataFrame that reports on the prevalence of a diseas/condition or the monthly
+        rate of certain events.
+        If multiple conditions in a module are defined, a pd.DataFrame must be returned with a column
```

Comment:
this part of the docstring doesn't seem right (left-over from one of the other docstrings?). Also, does it need to be a pd.Series that we're returning? I wonder if instead a simple and more flexible type (e.g. dict) would be suitable.